### ISSUE TITLE
PRIM_PROJECTOR (42) missing

### ISSUE THREAD

**Issue by raysilent** [raysilent/lslforge#62]
https://releasenotes.secondlife.com/simulator/2021-10-25.565008.html

**Comment by PellSmit** [comment]
Which branch should I make pull request?
I added PRIM_PROJECTOR and adapted to GHC-8.10.7.

**Comment by raysilent** [comment]
Hey Pell, I'm contemplating, may be making master branch the main place where recent changes are pushed and simply mark releases with tags? What do you suggest?

**Comment by PellSmit** [comment]
I agree to making master the latest branch, and we need to warn that debug functions are not maintained.
BTW, is  next version number 0.1.9.9?

**Comment by raysilent** [comment]
I merged 0.1.9.8 to master, so please proceed with a pull request to master as well. 0.1.9.9 is definitely fine!

**Comment by PellSmit** [comment]
Hi Ray, additionally I made big update:
1. Migrate native Haskell executable to GHC 8.10.7, we can generate Apple Silicon Native code.
2. Migrate Eclipse plugin to Java-11 platform, can use Eclipse 2021-12.
So I think it time to jump to new version number, like 0.2.0.0 or something.
How do you think?

You can try it on my repo's J-11_G-8.10.7 branch but version is still 0.1.9.8.
I built it on Eclipse RCP 2021-12 with Java 11.

**Comment by raysilent** [comment]
Awesome Pell, I made a #65 pull request. See if we are good to merge

**Comment by PellSmit** [comment]
Wait, I have not changed version number yet.
Which number is better, 0.1.9.9, 0.1.10.0 or 0.2.0.0?

**Comment by raysilent** [comment]
Let's do 0.1.9.9 and 0.2.0.0 when we release all the binaries

**Comment by PellSmit** [comment]
Ok, I pushed version number change commit but is it ok to use your master as a working branch? My branch includes only native plugin for macOS x86_64.

**Comment by raysilent** [comment]
You think it might mess up Eclipse updating?

**Comment by PellSmit** [comment]
At least we should update change log.
And I noticed another way to contribute Haskell plugin.
I think we can push lslforge-os-arch-version.jar file(including LSLForge binary),
it seems site.xml can build each plugin independently.
It needs inspections and descriptions in README.
So I think it's better working on 0.1.9.9 branch than working on master.
And merge it to master when it's ok.
Make a pull request to your 0.1.9.9 branch.

**Comment by raysilent** [comment]
Alright! Merged into 0.1.9.9. Which executables did you build?

**Comment by PellSmit** [comment]
I built only macosx_x86_64 executable and it is in eclipse/plugin/lslforge_macosx_x86_64_0.1.9.9.jar.
You can check it to unzip the jar file.
I don't have M1 mac and stack doesn't support aarch64 build yet,
I couldn't build aarch64 plugin.
But soon we can build aarch64 plugin by next version of stack.

**Comment by PellSmit** [comment]
Hi Ray, I added description to build native plugin. Try to build win32 plugin and correct if I made mistakes.

**Comment by ChloeConstantine** [comment]
Fixed in the version of Constants.hs I just uploaded
